integration/cluster_add_one_node_test.go

```diff
@@ -112,9 +117,9 @@ func TestClusterAddOneNode(t *testing.T) {
 		str   string
 		shard uint32
 	}{
-		{"foobarqux", 902},
-		{"bar", 397},
-		{"baz", 234},
+		{"jgfadfs", 10},
+		{"foobarqux", 2},
+		{"bar", 5},
```

Comment:
Ha - I knew this test was going to be susceptible to stuff like this, good job fixing these IDs too.